### ISSUE TITLE
helmfile with helm-gcs plugin gives error with"apply"

### ISSUE THREAD

**Issue by sarmak1179** [roboll/helmfile#958]
Hi Team,

Right now we are not able to use helmfile with helm-gcs plugin.
An issue has been raised with helm-gcs plugin at
https://github.com/hayorov/helm-gcs/issues/31

Failed to pull the images from the GCS repositories when using helmfile apply to deploy the helm charts:

Sample command that was generated:

exec: helm upgrade --install --reset-values sample-service-pre-qa sample-daily/sample-service --version ~1.2.84 --verify --wait --timeout 180 --atomic --namespace pre-qa --values /tmp/values205449575 --values /tmp/values081688986 --values /tmp/values212236337 --values /tmp/values592251868 --set replicas=1

err: release "sample-service-pre-qa" in "helmfile.yaml" failed: failed processing release sample-service-pre-qa: helm exited with status 1:
Error: storage: object doesn't exist
Usage:
helm-gcs pull gs://bucket/path [flags]

Flags:
-h, --help help for pull

Global Flags:
--debug activate debug
--service-account string service account to use for GCS

It works fine when we do a helm fetch.

Here are the versions used:

helm-gcs plugin version:

gcs 0.2.2 Manage repositories on Google Cloud Storage

helm version:

Client: &version.Version{SemVer:"v2.14.3", GitCommit:"0e7f3b6637f7af8fcfddb3d2941fcc7cbebb0085", GitTreeState:"clean"}

helmfile version:

helmfile version v0.90.5

Not sure if this is a problem with helmfile or with the plugin. Any help in this regard is appreciated

**Comment by mumoshu** [comment]
@sarmak1179 Hey! I'm not aware of any issue that may lead to this.

Curious, but does it work if you run `helm upgrade --install sample-daily/sample-service` directly?

**Comment by sarmak1179** [comment]
yes. helm plain upgrade works fine and helmfile lint works fine as well.
